exercises/grade-school/example.js

```diff
@@ -1,11 +1,11 @@
 module.exports = function School() {
   var db = {};
 
-  function add(student, grade) {
-    if (db[grade]) {
-      db[grade].push(student);
+  function add(student, gradeLvl) {
```

Comment:
Since we're making this change, let's make it totally readable: `gradeLevel` please!